server/lib/wsapi/lobbies.js

```diff
@@ -816,7 +806,7 @@ export class LobbyApi {
     })
   }
 
-  _onLoadingCanceled(lobby) {
+  _maybeCancelLoading(lobby, isLobbyEmpty) {
```

Comment:
probably default this new param to false since it isn't always passed? End result is probably the same but it's a bit more correct and will prevent some de-optimizations or something (doubt they matter very much here but whatever :) )